slides/segwit-transaction-malleability.tex

```diff
@@ -0,0 +1,351 @@
+			\item Consensus conflict! Who can spend the output?
+		\end{itemize}
+	}
+	\end{scriptsize}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{SegWit Transaction Types (P2WPKH)}
+	Pay-to-witness-public-key-hash (P2WPKH) is the SegWit equivalent to P2PKH.\\
+	\vspace{1.5em}
+	
+	\shadowbox{
+		\begin{minipage}[c]{4.1in}
+			\texttt{scriptSig:}\\
+			\texttt{scriptPubKey:\ 0 <pubKeyHash>}\\
+			\texttt{witness:\ <sig> <pubKey>}
+		\end{minipage}
+	}
+	\vspace{1.5em}
+
+	\textbf{Non-SegWit nodes:} output can be spent without the need for a signature.\\
+	\vspace{0.5em}
+	\textbf{SegWit nodes:} ''look for and verify the signature in the witness data."
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{SegWit Transaction Types (P2WSH)}
+	Pay-to-witness-script-has (P2WSH) is the SegWit equivalent to P2SH.\\
+	\vspace{1.5em}
+	
+	\shadowbox{
+		\begin{minipage}[c]{4.1in}
+			\texttt{scriptSig:}\\
+			\texttt{scriptPubKey:\ 0 <scriptHash256>}\\
+			\texttt{witness:\ Any valid script}
+		\end{minipage}
+	}
+	\vspace{1.5em}
+
+	\textbf{Non-SegWit nodes:} output can be spent by anyone.\\
+	\vspace{0.5em}
+	\textbf{SegWit nodes:} ''look for and verify the script in the witness data."
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{P2SH Embedded SegWit Transactions}
+
+	\begin{figure}
+		\input{../assets/figures/p2sh-segwit.tex}	
+	\end{figure}
+	\vspace{1.5em}
+	
+	\uncover<2->{
+		$\rightarrow$ The receiver creates a P2SH Bitcoin address embedding the SegWit script in the P2SH locking condition:
+	}
+	\vspace{1em}
+	
+	\uncover<3->{
+		\begin{itemize}
+			\item P2SH: \texttt{OP\_HASH160 <scriptHash> OP\_EQUALVERIFY}
+			\begin{itemize}
+				\item<4-> P2SH(P2WPKH): \texttt{<scriptHash> = 0 <pubKeyHash>}
+				\item<5-> P2SH(P2WSH): \texttt{<scriptHash> = 0 <scriptHash256>}
+			\end{itemize}
+		\end{itemize}
+	}
+	\end{frame}
+%%%
+
+%%%
+\begin{frame}{Native SegWit Address (Bech32)}
+	\begin{itemize}
+%		\item Native SegWit address format
+%		\item Always start with \texttt{bc1}
+%		\item Encode SegWit outputs in an efficient and secure way
+%		\item Case insensitivity
+%		\item Stronger checksums: Bose-Chaudhuri-Hocquenghem SOURCE
+		\item Consist of \texttt{bc1}, version number, witness script and six-character checksum	
+	\end{itemize}
+	
+	\vspace{1.5em}
+	
+	\begin{scriptsize}
+		\texttt{$\overline{BC}_{pkh}$ = bc1qryqazre3142dvj8jrg8ed9s6t56aql6puy8wk8}\\
+		\texttt{$BC_{sh}$ = bc1qrp33g0q5c5txsp9arysrx4k6zdkfs4nce4xj0gdcccefvpysxf3qccfmv3}
+	\end{scriptsize}
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Confirmation Capacity Increase}
+	\begin{scriptsize}
+	$S_{old}$: blocks can only be 1MB large.\\
+	$\rightarrow$ moving the redeem script to the witness field reduces the size of the transaction data.\\
+	\vspace{0.5em}
+	\uncover<2->{
+	$S_{new}$: blocks can only have a weight $W \leq 4$MB.\\}
+		\begin{columns}[T]
+			\begin{column}{0.5\textwidth}
+				\uncover<3->{
+					\begin{align*}
+						W &= 3 \cdot (1 - \beta) bs(\texttt{all})\\
```

Comment:
Add an explanation what "Beta" and "bs" is?
I think the derivation of the formula is not necessary but I don't understand how the "+1" occurs in the second line.